.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,20 @@
+// For format details, see https://aka.ms/devcontainer.json.
+{
+    "name": "Knative Serving",
+    "image": "gcr.io/knative-tests/test-infra/prow-tests:latest",
+
+    // Features to add to the dev container. More info: https://containers.dev/features.
+    "features": {
+        "ghcr.io/devcontainers/features/go:1": {}
```

Comment:
Suggestion: We can remove this `Go` Feature if the `prow-tests` image copies over `/usr/local/go` in [here](https://github.com/knative/test-infra/blob/main/images/prow-tests/Dockerfile#L166). This will make the (regular) dev container build two minutes faster.

Reply:
yeah we install go in the prow image

Reply:
Can we drop this feature - based on your comment we do not need it

Reply:
(This was true until last week) - We need it now because the `prow-test` image doesn't have `go`.
If you follow my suggestion, and once go is available in the image itself, then only we could remove this Feature.

Reply:
Looks like the dev-container just needs to run the following on start

`source "${HOME}/.gvm/scripts/gvm"`

Reply:
Nice, modified the config!